CardView: add Overview (#7454)
DevExpress/devextreme-documentation: concepts/05 UI Components/CardView/00 Overview/00 Overview.md

> DevExtreme CardView is a responsive grid control with various capabilities. The component supports data editing with validation, searching and filtering, layout customization, and more.

Reviewer: ```suggestion
DevExtreme CardView is a responsive grid that supports data editing with validation, search, filtering, layout customization, and more.
```
